pom.xml

```diff
@@ -50,6 +50,7 @@
 		<javaxMail.version>1.6.2</javaxMail.version>
 		<servletApi.version>3.0.1</servletApi.version>
 		<webServices.version>2.29.0</webServices.version>
+		<communications.version>1.2.0-SNAPSHOT</communications.version>
```

Comment:
NOTE: remember to merge changes onto communication omod and ensure that latest snapshot is released before you merge this.

Reply:
Sure